# HG changeset patch
# User Magnus Melin <mkmelin+mozilla@iki.fi>
# Date 1703181384 -3600
# Node ID 3fe32769e8a4fed3631f81aab79f60e8fdc15fd2
# Parent  3844eb9632d9dcfbe61e86785f60958093d6523c
Bug 1870809 - Workaround to ensure OpenPGP alerts have some minimum size. r=aleca

This is a workaround really intended for 115 only. The code in question is getting removed, and fixing the sizing issues is difficult without completely rewriting the html...

Differential Revision: https://phabricator.services.mozilla.com/D196947

diff --git a/mail/extensions/openpgp/content/ui/enigmailMsgBox.js b/mail/extensions/openpgp/content/ui/enigmailMsgBox.js
--- a/mail/extensions/openpgp/content/ui/enigmailMsgBox.js
+++ b/mail/extensions/openpgp/content/ui/enigmailMsgBox.js
@@ -6,16 +6,17 @@
 
 "use strict";
 
 var { AppConstants } = ChromeUtils.importESModule(
   "resource://gre/modules/AppConstants.sys.mjs"
 );
 
 function onLoad() {
+  document.documentElement.style.minHeight = "120px";
   var dlg = document.getElementById("enigmailMsgBox");
   dlg.getButton("cancel").setAttribute("hidden", "true");
   dlg.getButton("extra1").setAttribute("hidden", "true");
   dlg.getButton("extra2").setAttribute("hidden", "true");
 
   document.getElementById("filler").style.maxWidth =
     screen.availWidth - 50 + "px";
 
